install.cmd

```diff
@@ -108,18 +108,30 @@ SET COMMANDS_WITHOUT_REPO=git-alias git-extras git-fork git-setup
 
 echo Installing binaries...
 FOR /R "%GITEXTRAS%\bin" %%i in (*.*) DO (
+::  Fixation for Windows 10.0.17134 Build 17134 (Won't install without /E)    
+::  Echo file being written for testing
```

Comment:
IMHO, these two comments don't look like necessary. The first one could be put just before `MORE` command, and the second one is too straightforward.